test/oli/groups_test.exs

```diff
@@ -67,15 +67,25 @@ defmodule Oli.GroupsTest do
       assert error =~ "has already been taken"
     end
 
-    test "delete_community/1 deletes the community" do
+    test "delete_community/1 marks the community as deleted" do
       community = insert(:community)
-      assert {:ok, %Community{}} = Groups.delete_community(community)
-      refute Groups.get_community(community.id)
+      assert {:ok, deleted_community} = Groups.delete_community(community)
+      assert deleted_community.status == :deleted
     end
 
     test "change_community/1 returns a community changeset" do
       community = insert(:community)
       assert %Ecto.Changeset{} = Groups.change_community(community)
     end
+
+    test "search_communities/1 returns all communities meeting the criteria" do
+      active_communities = insert_pair(:community, status: :active)
+      _deleted_community = insert(:community, status: :deleted)
```

Comment:
NIT: remove the variable

Reply:
Actually I assigned this underscored variable to be more explicit about what type of communities we are creating here, even though we don't use it later.
Anyway, I think the insert itself is explicit enough so I'll remove it.